packages/frontend/src/components/svg/GenerateNewIcon.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default () => {
```

Comment:
Can we make this a raw `.svg` file instead of a JS component? It'll be cached that way and it's not being customized in any way through JS here

Reply:
I'd prefer to keep it a component so that there's no initial flicker on the 'generate new' button on the seed phrase page. Even if it's fast, it doesn't look great with that initial load. I agree about the caching etc and do that with larger files or graphics, but since this icon is so tiny, the performance vs perceived performance here seems user will perceive it as suboptimal UI if we don't show it instantly